vmdb/config/event_handling.tmpl.yml

```diff
@@ -1370,14 +1370,14 @@ event_handling:
   - refresh:
     - ems
   AWS_EC2_Instance_CREATE:
-  - policy:
-    - src_vm
-    - vm_create
```

Comment:
Most create events have a comment in the file like this to document the behavior.
`# Ignore <provider_create_event> for policy for now the vm_create event will be raised when the VM is created`

The one exception is the `MarkAsVirtualMachine` event since the template is already in the system and being converted over to a VM.